pkg/cloud/services/interfaces.go

```diff
@@ -39,6 +39,6 @@ type EC2MachineInterface interface {
 // SecretsManagerInterface encapsulated the methods exposed to the
 // machine actuator
 type SecretsManagerInterface interface {
-	Delete(m *scope.MachineScope) error
+	Delete(prefix string, chunkCount int32) error
```

Comment:
Same comment, is this a style preference? From an interface perspective, I kinda preferred the old way, but not an issue